action.yaml

```diff
@@ -164,14 +164,29 @@ runs:
         if [ "${{ inputs.verbose }}" == "true" ]; then
           extraArgs+=("--verbose")
         fi
+        touch /tmp/console.log
         sudo /bin/lvh run --host-mount=${{ inputs.host-mount }} --image /_images/${{ inputs.test-name }}.qcow2 \
             --daemonize -p ${{ inputs.ssh-port }}:22 --serial-port ${{ inputs.serial-port }} \
             --cpu=${{ inputs.cpu }} --mem=${{ inputs.mem }} --cpu-kind ${{ inputs.cpu-kind }} \
+            --console-log-file /tmp/console.log \
             ${extraArgs[@]}
 
     - name: Wait for VM's SSH Server
       shell: bash
       run: |
+        n=0
+        until [ "$n" -ge 30 ]; do
+          started=1
+          if grep -E ".*OK.*Started.*ssh.*" /tmp/console.log; then
+            break || started=0
+          elif grep -E ".*FAILED.*Failed.*to.*start.*ssh*" /tmp/console.log; then
```

Comment:
Maybe let's dump /tmp/console.log in this case? The same after 30 unsuccessful sleeps.

Reply:
Ah good idea. What would be the best way to do this? I'm assuming new can just `cat` the log, or does it need to be saved as an artifact?

Reply:
Yeah, cat should do the job